test/image.jl

```diff
@@ -8,17 +8,29 @@ mutable struct MyNeuralNetwork <: MLJFlux.Builder
     kernel2
 end
 
-function MLJFlux.build(model::MyNeuralNetwork, rng, ip, op, n_channels)
+# to get a matrix whose last dimension mathces that of the array input (the batch size):
+function make2d(x)
+    l = length(x)
+    b = size(x)[end]
+    reshape(x, div(l, b), b)
```

Comment:
So I thought this use of `reshape` would avoid the "scalar indexing error", but tests are still not passing.